Core/include/Acts/EventData/MultiComponentTrackParameters.hpp

```diff
@@ -239,4 +209,75 @@ class MultiComponentBoundTrackParameters {
   }
 };
 
+/// This class mimics the behaviour of the curvilinear parameters for ordinary
```

Comment:
So do you think there's a better way to generalize this?

Reply:
hmm I though if it might be better to construct these parameters with a perigee surface then a plane surface... But then its kind of the same as with the target propagation...

But this thing is not used anywhere currently, and the the change that somebody uses the multistepper outside the GSF is pretty low currently, right? So maybe it doesn't matter...